scripts/openshiftci-presubmit-unittests.sh

```diff
@@ -14,8 +14,12 @@ export GOLANGCI_LINT_CACHE="/tmp/.cache"
 
 make goget-tools
 make validate
-make test
+GOFLAGS='' make test
```

Comment:
GOFLAGS=-mod-vendor is already set in CI container and this conflicts with -mod=vendor param